packages/plugin-loyalties-api/src/graphql/resolvers/queries/scoreCampaign.ts

```
@@ -0,0 +1,115 @@
+    { _id },
+    { models, subdomain }: IContext
+  ) {
+    const services = await getServices();
+
+    let attributes: any[] = [];
+
+    for (const serviceName of services) {
+      const service = await getService(serviceName);
+      const meta = service.config?.meta || {};
+
+      if (meta && meta?.loyalties && meta?.loyalties?.aviableAttributes) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)